### DISCUSSION TITLE
New AUR / Arch Linux package

### DISCUSSION THREAD

**Discussion by alerque** [jotaen/klog#280] [Show and tell]
Are the website sources available somewhere to submit updates to the copy there? I wanted to update the Arch Linux packaging to note the [package I setup built from sources](https://aur.archlinux.org/packages/klog-time-tracker) in addition to the -bin variant you already mention. Also there is a Nix package available that could be mentioned too.

**Comment by jotaen** [comment]
Cool, thanks for submitting this! I’ve added your AUR package to the website (the website is source isn’t public).

Thanks for hinting at the Nix package, I’ve [reached out to the maintainer about it](https://github.com/NixOS/nixpkgs/pull/270583#issuecomment-1899292643).
